docs/catalyst_101.md

```diff
@@ -136,6 +136,159 @@ Once the existing members have voted to offer you membership, it’s up to you t
+
+#### Only pull scoped, prioritized tickets into the sprint
+
+Goal: make it easier to decide what to work on by agreeing on priority as a team and providing sufficient context for team members to get started on each item.
+
+Activity:
+
+Use the mid sprint checkin as a parallel work block to pull items into Backlog - aiming for at least 2N items in Backlog, where N is the number of people on the team. All items in Backlog should have scope and a next step:
+
+- scope: when would we consider this issue to be done? what criteria have to be met, or what behavior has to exist?
+- next step: what's the first step to getting this working?
+
+In the sprint review, instead of discussing status for each item in the sprint, prioritize the tickets that are in Backlog according to the following schema:
+
+- "Urgent" -> blocking work on high-priority tickets
+- "High" -> if we do not finish this during the next two weeks, we will not hit our quarterly goals
+- "Medium" -> we'd like to finish this during the next two weeks
+- "Low" -> if there's not other stuff to do, it's worth knocking out
+
+In the sprint planning:
+
+- Pull high priority items from in-flight and backlog into the sprint until it feels like we have enough, but not too much, work.
+  - If a low priority item is in flight, we should bump it for higher-priority work.
+
+Outside of the meetings:
+
+- If a new issue comes up, it goes in the Icebox unless there's a compelling case for it to be Urgent, in which case we talk about it as a team and pull it into the sprint if it's truly Urgent.
+
+Success criteria:
+
+- it feels easier than before to start work on a new ticket when your old ticket is done
+- minimal time spent doing homework outside of meetings
+
+Accountability/how we'll stick to it:
+
+- parallel work blocks are more engaging than "everyone talk through everything in series."
+
+#### Limit work-in-progress
+
+Goal: reduce overwhelm from "too many things going on at once" and identify bottlenecks in our process
+
+Activity:
+When moving work through the pipeline to Done:
+
+- If there are more than N+1 "things" in the next phase (N = team size) do not move your work to the next phase. Instead, find a thing in the next phase and try to move it - i.e. review someone's pull request, or see if you can help someone get their code into a reviewable state.
+  - one "thing" is: a PR, an issue, or a linked pair of issue + PR (which counts as one "thing").
+
+During mid sprint check-in and sprint review:
+
+- Thing Counting Phase at the beginning of the meeting where we count the Things and make sure we're not over the limit.
+
+Outside of meetings:
+
+- Say no to new things if they would push you over WIP limits.
+
+Success criteria:
+
+- people work together on projects more
+- people have fewer things going on at once
+
+Accountability/how we'll stick to it:
+
+- Just the in-meeting Thing Counting Phase. Maybe we can make it a race to make it engaging.
```

Comment:
Is there any kind of github automation we could do to enforce this automatically? And if so, would implementing that automation be overly complicated?

Reply:
I think we could do some sort of script that cleans things up via the API but I don't think we can enforce a WIP limit within the UI.